deploy/base/kustomization.yaml

```diff
@@ -27,6 +27,10 @@ patchesStrategicMerge:
                 value: gcr.io/k8s-staging-sp-operator/security-profiles-operator:latest
               - name: RELATED_IMAGE_NON_ROOT_ENABLER
                 value: bash:5.0
+              - name: RELATED_IMAGE_POLICIES_COPIER
+                value: quay.io/jaosorior/udica
+              - name: RELATED_IMAGE_SELINUXD
+                value: quay.io/jaosorior/selinuxd
```

Comment:
let's unify this by adding the `udica` dependency to the `selinuxd` image. This way we don't need the extra image.

Reply:
I'm not sure I understand what you are proposing. Copy the files from `udica` to the `selinuxd` image as part of the image build process and then use `selinuxd` as the init container that primes the selinux drop-directory?

Reply:
in the selinuxd Dockerfile:
```
microdnf install -y udica
```
That would basically install the base policies that `udica` comes with.
Then we could use that same image as the init container that primes the drop-directory.

Reply:
oh, right. Packages still exist :-)

Reply:
Done in selinuxd.
Here's a follow-up question: I think we can ditch the second init container entirely and change the ownership of `/etc/selinux.d` in the init container. Then we could copy the files into the directory when selinuxd starts with a lifecycle handler. It is maybe a bit hacky to have the logic split over two places, but at least it would allow us to keep the DS template more minimal and better manageable.

Reply:
As discussed on Slack, let's do this if/when we need to make the DS manageable better. I added a comment in the meantime.